Formula/m/monero.rb

```diff
@@ -3,10 +3,9 @@ class Monero < Formula
   homepage "https://www.getmonero.org/"
   # TODO: Check if we can use unversioned `protobuf` at version bump
```

Comment:
Was this checked?

Reply:
not really, let me check it now

Reply:
protobuf24 works for me (build/test) in my local